startReactCommerce.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
 
+if [ -n "${algoliaApiKey+set}" ]; then
+    echo "Starting react-commerce localdev"
+else
+    echo "You must set the algoliaApiKey variable in your environment."
+    echo -e 'Try running with: \n\nexport algoliaApiKey=YOUR_API_KEY \nsource startReactCommerce.sh\n'
+    echo -e 'Or you can run it in one line with: \n\nalgoliaApiKey=xyz123 source startReactCommerce.sh'
```

Comment:
Does this need to be `source`? I would have expected just `./startReactCommerce.sh` or `bash startReactCommerce.sh` or something.

Reply:
`./` works fine with the correct permissions. Updated in [2a97727](https://github.com/iFixit/react-commerce/pull/1831/commits/2a977274ff27d127d6f8e5c683d68cb6635087b3)